lnwire/single_funding_open_proof.go

```diff
@@ -9,7 +9,8 @@ import (
 // responder after the previously constructed funding transaction has achieved
 // a sufficient number of confirmations. It is the initiator's duty to present
 // a proof of an open channel to the responder. Otherwise, responding node may
-// be vulernable to a resource exhasution attack wherein the a requesting node
+// be vulnerable to a resource exhaustion attack wherein the a requesting
+// node
```

Comment:
Can you fix the character folding below this? Within the codebase comments are typically limited to 80 characters.

Thanks!